main.go

```diff
@@ -186,10 +191,11 @@ func main() {
 	// ********************************************************************************
 	log.FromContext(ctx).Infof("executing phase 3: creating icmp server ipam")
 	// ********************************************************************************
-	_, ipnet, err := net.ParseCIDR(config.CidrPrefix)
+	ipamChain, err := getIPAMChain(config.CidrPrefix, config.Ipv6Prefix)
```

Comment:
Could we simply have this take a list similar to how we handle it in cmd-forwarder-vpp ?

Reply:
@edwarnicke Sure, technically is not a big deal. 

The reason to have this additional parameter is to be aligned with the cmd-nse-remote-vlan, where it is already used for dual-stack.
https://github.com/networkservicemesh/cmd-nse-remote-vlan/blob/51251b32c2c6be7f9f943dac4de17be9c5188f45/internal/pkg/config/config.go#L45

To deprecate that one would be a not backward compatible change. Maybe I should add the possibility on cmd-nse-remote-vlan to have a list in CidrPrefix parameter also and leave the Ipv6Prefix as is.

Reply:
Its also possible I'm just weird as I simply see dual stack as 'We have more than one IP, and they happen to be from different families'... :)